File: Astronomy-objc/Astronomy-objc/RoverPhotoCollectionViewCell.m
//
//  RoverPhotoCollectionViewCell.m
//  Astronomy-objc
//
//  Created by Dillon P on 4/28/20.
//  Copyright © 2020 Dillon's Lambda Team. All rights reserved.
//

#import "RoverPhotoCollectionViewCell.h"

@implementation RoverPhotoCollectionViewCell

- (void)prepareForReuse
{
    self.roverImage.image = [UIImage imageNamed:@"roverImagePlaceholder"];
    
    [super prepareForReuse];
}

@end

File: Astronomy-ObSwiftive-c/Astronomy-ObSwiftive-c/View Controllers/RoverPhotoCollectionViewCell.swift
//
//  RoverPhotoCollectionViewCell.swift
//  Astronomy-ObSwiftive-c
//
//  Created by Craig Swanson on 4/27/20.
//  Copyright © 2020 craigswanson. All rights reserved.
//

import UIKit

class RoverPhotoCollectionViewCell: UICollectionViewCell {
    

    override func prepareForReuse() {
        super.prepareForReuse()
        roverPhotoImageView.image = #imageLiteral(resourceName: "MarsPlaceholder")
    }
    
    @IBOutlet var roverPhotoImageView: UIImageView!
    
}
